src/pages/changelog/2025-04-03.mdx

```diff
@@ -9,6 +9,6 @@ We have introduced a new mutation that enables changing a signature order after
 Initially it supports changing maxSignatories until the first signatory has signed.
 Please reach out if you have other needs for modifying signature orders post creation.
 
-[An example](http://localhost:8000/signatures/graphql/examples/#changing-signature-order) can be found in the Signatures documentation.
+[An example](https://docs.criipto.com/signatures/graphql/examples/#changing-signature-order) can be found in the Signatures documentation.
```

Comment:
Use `/signatures/graphql/examples/#changing-signature-order` so it also works for deploy previews etc